.github/workflows/linux-pr-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: GitHub Actions CI - Linux
```

Comment:
Can this eventually be included in `tests.yml`, as part of the strategy matrix? We should try to aim for that, and probably make the code here as similar as possible to make easy to integrate (though I suspect you've already done this).

Reply:
Ye, we can, but then we will have to conditionally run every command for Linux, `if Linux then run docker...brew...`, like that. I mean, this approach IMO would be the best here, because we will end up with one workflow for tests, less code duplication and actual control of jobs dependency, as now Linux tests will run regardless of tap_syntax job result.

Reply:
Those sound like good reasons to me.